.github/workflows/update-npm-deps.yml

```diff
@@ -0,0 +1,44 @@
+name: Update npm dependencies
+
+on:
+  schedule:
+    - cron: '0 8 * * 1'
+
+jobs:
+  update_deps:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup node version
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.3.0
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Update dependencies
+        run: npm run update-deps
```

Comment:
This action is going to fail, as the `update-deps` npm script hasn't been added to the `package.json` in this repo!